### ISSUE TITLE
Hardcoded attribute title="file input" (should be removed)

### ISSUE THREAD

**Issue by alphestdev** [FineUploader/fine-uploader#1526]
FineUploader 5.5.0
fine-uploader.js
Line 1240

`input.setAttribute("title", "file input");`

All text should come from the template. Hardcoded text is not suitable, especially for multi-language environments.

**Comment by rnicholus** [comment]
Perhaps a title on a containing element that exists in the template will be sufficient. If not, that attribute can be transferred programmatically to the file input.

**Comment by rnicholus** [comment]
Moving this to 5.6.0, and I'll start work on this at once.

**Comment by rnicholus** [comment]
In order to properly address this, I'll need to add a new [`text` option](http://docs.fineuploader.com/branch/master/api/options.html#text), such as `text.fileInputTitle`. Here, you will be able to optionally specify a `title` attribute to be used when Fine Uploader created the `<input type="file">`. 

You can't specify this yourself on the file input (easily) since it is removed and re-created whenever files are submitted to the uploader. This is the only way to "reset" the file input element. Another option would be to leave the `title` off of the file input and specify one on the parent element instead (such as the one in the template), but this will not work cross-browser. For example, Firefox defaults to the native hover tooltip when the file input `title` attribute is omitted.

**Comment by deviprsd21** [comment]
That is the best way so far, of course, I found it really odd in my design to have the popup with a message not desired. It was irritating. So, I had a quick workaround 
```javascript
$("#fine-uploader").fineUploader({
    //options
}).on('complete', function(e, id, name, response){
    $(e.currentTarget).find('input[name="qqfile"]').attr('title', '');
}).find('input[name="qqfile"]').attr('title', '');
```

Like you said each time it was created and removed, so after each request completion I had to reset it again.

**Comment by alphestdev** [comment]
@deviprsd21 I did the same thing using `onStatusChange` to remove attribute as early as possible but still appeared for a short time and could be visible when hovered. I ended up removing the line from the fine-uploader source as I don't use tooltip text on a button that already has text.

**Comment by deviprsd21** [comment]
Yup, anyway it has been added to the milestone for 5.6.0, all we have to do is wait.

**Comment by rnicholus** [comment]
I'm starting work on this feature this week, and once it is completed and verified by someone in this thread, I'll release 5.6.0.

**Comment by deviprsd21** [comment]
Oh well, great :+1:

**Comment by rnicholus** [comment]
This has been completed and is part of build 2 of 5.6.0. Please verify that this solves your issue. If it does, I'll formally release this as part of 5.6.0 next. You can download this pre-release build at https://s3.amazonaws.com/releases.fineuploader.com/develop/5.6.0-1/fine-uploader-5.6.0-2.zip.

**Comment by rnicholus** [comment]
Note that I added a [`text.fileInputTitle` option](http://docs.fineuploader.com/branch/develop/api/options.html#text.fileInputTitle), and the [`extraButtons` option](http://docs.fineuploader.com/branch/develop/api/options.html#extraButtons.fileInputTitle) also accepts a `fileInputTitle`.

**Comment by alphestdev** [comment]
Tested 5.6.0-2. It works when explicitly defined as empty string. If the option is not set, it still appears as `"file input" `.

**Comment by rnicholus** [comment]
Yes, this is the expected behavior. I kept the default value as I try to avoid any unnecessary changes in default behavior outside of a major version release.

**Comment by deviprsd21** [comment]
@alphestdev that should be the normal behavior. You can set it as nothing and it won't appear I guess. Will try now.

**Comment by rnicholus** [comment]
There is no jquery version. Fine Uploader offers a thin jquery wrapper around the library, but the wrapper provides no real value (#1310).

**Comment by rnicholus** [comment]
@deviprsd21 Look good on your end? If so, I'll release 5.6.0 today.

**Comment by deviprsd21** [comment]
Works like charm :heart_eyes:

**Comment by rnicholus** [comment]
This has been released as part of 5.6.0.
